### ISSUE TITLE
Hard shutdown after trying to Turn on FPS counter with L3 + R3

### ISSUE THREAD

**Issue by WaggishZX** [aldostools/webMAN-MOD#936]
A notification does show up when doing the shortcut, but instantly after that the console hard shuts down.
This was not an issue with the previous release.

(PS3 Slim CECH-2501A on 4.90 w/HEN)

**Comment by aldostools** [comment]
Please try using the online updater found in webMAN MOD Setup to get the latest build. Then reboot after install.

Or install manually the VshFpsCounter plugin in `/dev_hdd0/tmp/wm_res`
https://github.com/TheRouletteBoi/RouLetteVshMenu/releases/tag/fps-counter-2.7

If the problem persists after re-enable HEN, report the issue to the author of VshFpsCounter => @TheRouletteBoi

**Comment by TheRouletteBoi** [comment]
oh now that he mentioned a instant shutdown it maybe becuase sprx wasn't signed using 3.55 keys. I will add a automation process for prx signing when building the project

**Comment by TheRouletteBoi** [comment]
@aldostools have you tried using [SecureTool.exe by xorloser](https://www.psx-place.com/threads/release-securetool-exe-by-xorloser.39367/)?
So far 2 people have reported to me that sprx's used with SecureTool.exe simply don't load on ps3

This is how I've used it
`SecureTool.exe -c 1 -k APP00A -ms VshFpsCounter.prx VshFpsCounter.sprx`

**Comment by aldostools** [comment]
> have you tried using [SecureTool.exe by xorloser](https://www.psx-place.com/threads/release-securetool-exe-by-xorloser.39367/)? 

No, I use [scetool.exe](https://github.com/aldostools/webMAN-MOD/blob/master/scetool.exe). It requires the keys in the folder [data](https://github.com/aldostools/webMAN-MOD/tree/master/data)

````
scetool -0 SELF -1 TRUE -s FALSE -2 0A -3 1070000052000001 -4 01000002 -5 APP -6 0003004000000000 -A 0001000000000000 --self-ctrl-flags 4000000000000000000000000000000000000000000000000000000000000002 -e webftp_server.prx webftp_server.sprx
````

Check the [makefile](https://github.com/aldostools/webMAN-MOD/blob/master/makefile#L55)

**Comment by TheRouletteBoi** [comment]
> > have you tried using [SecureTool.exe by xorloser](https://www.psx-place.com/threads/release-securetool-exe-by-xorloser.39367/)?
> 
> No, I use [scetool.exe](https://github.com/aldostools/webMAN-MOD/blob/master/scetool.exe). It requires the keys in the folder [data](https://github.com/aldostools/webMAN-MOD/tree/master/data)
> 
> ```
> scetool -0 SELF -1 TRUE -s FALSE -2 0A -3 1070000052000001 -4 01000002 -5 APP -6 0003004000000000 -A 0001000000000000 --self-ctrl-flags 4000000000000000000000000000000000000000000000000000000000000002 -e webftp_server.prx webftp_server.sprx
> ```
> 
> Check the [makefile](https://github.com/aldostools/webMAN-MOD/blob/master/makefile#L55)

thanks, I used to manually sign the prx using TrueAncestor SELF Resigner. I made a pull request #937 to automatically sign prx upon build.
Here is the updated sprx file with proper signing [Download](https://gofile.io/d/BfZc74)
